### PR TITLE
docs: Fix README

### PR DESCRIPTION
**Pull request by assynour**
Please check if the readme is clear

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,51 @@
 # bpmn-visualization 'getting started' tutorial
 
-A getting started demo that uses bpmn-visualization library to visualize runtime monitoring data.
+A getting started demo that uses bpmn-visualization library to visualize runtime monitoring data ⏲️.
```
Comment (tbouffard) [inline README.md:3]:
```suggestion
A getting started demo that uses the [bpmn-visualization](https://github.com/process-analytics/bpmn-visualization-js) library to visualize runtime monitoring data ⏲️.

```

File: README.md [hunk 2]
```diff
@@ -1,57 +1,51 @@
...
+- State of instances: whether or not they predefined KPIs ⏱️ are violated:
+    - Two KPIs are considered: completion and waiting time of activities.
+    - Two levels of violation are defined: risky 🟠 and criticial 🔴.
 
-The following visualization is used:
+## Visualization
 
-**Overlays on activities:**
+### Running instances and their number ⚙️
+- **animated activity stroke**: to indicate that there are running instances of the corresponding activitiy
```
Comment (tbouffard) [inline README.md:15]:
```suggestion
- **animated activity stroke**: to indicate that there are running instances of the corresponding activity

```

File: README.md [hunk 3]
```diff
@@ -1,57 +1,51 @@
...
+- 🟢 Green: no violation
+- 🟠 Orange: violation with a risky level
+- 🔴 Red: Violation with a critical level
 
-- green: avg execution time of instances is “efficient” according to a predefined KPI (throuhput time)
-- orange: avg execution time is “ineffcient with a risky level” according to a predefined KPI (throuhput time)
-- red: avg execution time is “ineffcient with a criticial level” according to a predefined KPI (throuhput time)
+#### Activities:
+- **Overlay filling color**: 
+    - Green: number of instances running on time according to a predefined completion time KPI
```
Comment (tbouffard) [inline README.md:27]: ℹ️ you may add the emoji for colors as you did before

File: README.md [hunk 4]
```diff
@@ -1,57 +1,51 @@
...
+- **Speed of the animated stroke:**  The following is applied in order:
+    - slow: if there exist instances running late with a critical level.
+    - medium: if there exist instances running late with a risky level
+    - fast: in case all instances are running on time
 
-- red if at least one instance is running critically late
-- else orange if at least one instance is running risky late
-- else green (meaning that it’s still on time)
+- **Shadow:** The following is applied in order:
+    - Red: if there exist instances running late with a critical level.
```
Comment (tbouffard) [inline README.md:37]: ℹ️ you may add the emoji for colors as you did before

File: README.md (hunk 3 above)
Comment (assynour) [reply README.md]: fixed in bdebdea1f292fcaeec621bbf12e0f15e3a38aad3

File: README.md (hunk 4 above)
Comment (assynour) [reply README.md]: fixed in bdebdea1f292fcaeec621bbf12e0f15e3a38aad3

**Pull request merged**